site/pages/create-consequence-services/[disruptionId]/[consequenceIndex].page.tsx

```diff
@@ -54,15 +54,43 @@ const filterConfig = {
     matchFrom: "any" as const,
 };
 
-export const fetchStops = async (serviceId: number): Promise<Stop[]> => {
+export const fetchStops = async (serviceId: number, vehicleMode?: string): Promise<Stop[]> => {
```

Comment:
vehicleMode could be of type VehicleMode instead of string

Reply:
done